ext/afform/core/Civi/Api4/Afform.php

```diff
@@ -298,6 +298,15 @@ public static function getFields($checkPermissions = TRUE) {
           'data_type' => 'Timestamp',
           'readonly' => TRUE,
         ],
+        [
+          'name' => 'confirmation_type',
+          'data_type' => 'Integer',
```

Comment:
This field needs a pseudoconstant callback so that 1 and 2 can be resolved to names/labels by the API.

Reply:
Ok, I have made the changes.